tensorflow_model_optimization/python/core/clustering/keras/cluster.py

```diff
@@ -14,12 +14,11 @@
 # ==============================================================================
 """Clustering API functions for Keras models."""
 
-from tensorflow.python import keras
-from tensorflow.python.keras import backend as k
-from tensorflow.python.keras import initializers
-from tensorflow.python.keras.engine.base_layer import Layer
-from tensorflow.python.keras.engine.input_layer import InputLayer
-from tensorflow.python.keras.utils.generic_utils import custom_object_scope
+from tensorflow import keras
+from tensorflow.keras import backend as k
+from tensorflow.keras import initializers
+from tensorflow.keras.layers import Layer, InputLayer
```

Comment:
Should we be importing modules, instead of classes, as outlined in [Google Python Style Guide](https://github.com/google/styleguide/blob/gh-pages/pyguide.md)? I guess we can use aliases similar to what is done in [pruning_wrapper.py](https://github.com/tensorflow/model-optimization/blob/ccf6e98cc7ac847a7eaab4068b23139a63b32a58/tensorflow_model_optimization/python/core/sparsity/keras/pruning_wrapper.py#L36), if needed.

Reply:
> Use import statements for packages and modules only, not for individual classes or functions.

I agree, we clearly seem to be in violation of this rule here. I will update the imports in consistence with the style guide and the pruning code.